index.html

```diff
@@ -7,10 +7,25 @@
   <title>Sprint Challenge - Home</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
```

Comment:
Be careful here, when I downloaded your project none of the CSS was working.  The path to the CSS file is relative to where this file lives.  Where you have it now it thinks the index.css lives in the same folder that the HTML file lives, but it doesn't, it lives in css\index.css.